lib/kioskPrinter.js

```diff
@@ -66,50 +52,75 @@ function printWallet (wallet, printerCfg) {
   })
 }
 
+function checkStatusZebra (printerCfg) {
+  return new Promise((resolve, reject) => {
```

Comment:
Don't need to create a new Promise here since openSerialPort creates a promise we can chain off of.

Reply:
Can you elaborate on your idea please? I don't see how we could chain the events given that we'll need to wait on the `'data'` event to give us the printer status and then call `resolve` to continue the execution on the calling functions.

Reply:
Yeah, since you're monitoring ``data`` in the end it's not that simple. I think we can keep it this way.